.github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts

```diff
@@ -40,68 +35,23 @@ async function createHealthStatusConfig() {
     github: {
       organization: GITHUB_ORG,
       repository: GITHUB_REPO,
-      defaultBranch: 'master',
+      defaultBranch,
       workflows: [
         {
           name: 'optimize-ci',
           branches: ciBranches,
         },
-        {
-          name: 'ci',
-          branches: maintenanceBranches,
-        },
         'optimize-zeebe-compatibility',
-        'optimize-java-compatibility',
-        'optimize-engine-compatibility',
         'optimize-es-compatibility',
         'optimize-os-compatibility',
-        'optimize-upgrade-data-performance',
-        'optimize-import-dynamic-data-performance',
-        'optimize-import-static-data-performance',
-        'optimize-import-mediator-permutation',
-        'optimize-history-cleanup-performance',
-        'optimize-connect-to-secured-es',
-        'optimize-cluster-test',
-        'optimize-e2e-tests',
-        'optimize-release-optimize',
-        'optimize-trivy-check',
-      ],
-    },
-    githubPrs: {
```

Comment:
I don't think there is a list of deps PRs only related to Optimize as we are providing those updates for the whole monorepo, so I delete this whole section